packages/shared/lib/core/network/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './enums'
```

Comment:
These should be alphabetized.

Reply:
There was a few issues with some types / enums used with each other not exporting in the correct order